src/my_project/tagging/metadata_controller.py

```
@@ -381,19 +387,21 @@ def move_to_publisher_folder(self, new_name: str, publisher_int: int) -> None:
             new_name (str): The name of the comic archive that was decided from metadata.
             publisher_int (int): The unique ID of the publisher, these align with the database ID's.
         """
-        for subdir in ROOT_DIR.iterdir():
+        for subdir in self.config_manager.config.comicsroot.path.iterdir():
             if subdir.is_dir() and subdir.name.startswith(str(publisher_int)):
                 new_path = subdir / new_name
                 shutil.move(self.original_filepath, new_path)
-                logging.info(f"Moved file to {subdir.name}")
+                logger.info(f"Moved file to {subdir.name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)